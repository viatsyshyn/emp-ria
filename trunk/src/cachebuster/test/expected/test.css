body {
  background: url(test/fixtures/test.l2g13b.gif);
}