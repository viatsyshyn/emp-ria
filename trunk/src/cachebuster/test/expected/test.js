function test() {
  var myGif = 'test/fixtures/test.l2g13b.gif';
}