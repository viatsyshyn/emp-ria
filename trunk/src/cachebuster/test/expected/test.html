<!DOCTYPE html>
<html>
<head>
    <title></title>
    <link href="test/fixtures/test.74kbgu.css" type="text/css" rel="stylesheet" />
    <script src="test/fixtures/test.9qyb4a.js" type="text/javascript"></script>
</head>
<body>
    <img src="test/fixtures/test.l2g13b.gif"/>
</body>
</html>